app/controllers/admin/settings/billing.js

```diff
@@ -0,0 +1,21 @@
+import Controller from '@ember/controller';
+import { timezones } from 'open-event-frontend/utils/dictionary/date-time';
+import { computed } from '@ember/object';
+
+export default Controller.extend({
+  timezones,
+  adminBillingEmail: computed('authManager.currentUser', function() {
```

Comment:
why are you using a computed property for this?

Reply:
@CosmicCoder96 When I use it without a computed property, it shows me that it's undefined `undefined is not an object (evaluating '(void 0).authManager')`.

Reply:
in the server schema, didn't you specify a separate field mail for admin billing? Why do you need this one?